include/amici/steadystateproblem.h

```diff
@@ -143,22 +162,25 @@ class SteadystateProblem {
      * @param xdot current rhs
      * @param atol absolute tolerance
      * @param rtol relative tolerance
+     * @param ewt error weight vector
      * @return root-mean-square norm
      */
     realtype getWrmsNorm(AmiVector const &x,
                          AmiVector const &xdot,
                          realtype atol,
-                         realtype rtol);
+                         realtype rtol,
+                         AmiVector &ewt);
```

Comment:
If we have quadratures backward, the ewt-vector will have another size. this is why we now need to pass it to getWrmsNorm

Reply:
If you feel like its worthwhile mentioning this in the PR, it probably makes sense to have this as comment in the code. I added some suggestions where I feel like this should be documented.